samples/copy-from-container/pom.xml

```
@@ -0,0 +1,89 @@
+<project xmlns = "http://maven.apache.org/POM/4.0.0"
+   xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+   xsi:schemaLocation = "http://maven.apache.org/POM/4.0.0
+   http://maven.apache.org/xsd/maven-4.0.0.xsd">
+   <parent>
+      <groupId>io.fabric8.dmp.samples</groupId>
+      <artifactId>dmp-sample-parent</artifactId>
+      <version>0.34-SNAPSHOT</version>
+      <relativePath>../pom.xml</relativePath>
+   </parent>
+   <modelVersion>4.0.0</modelVersion>
+   <artifactId>dmp-sample-copy-from-container</artifactId>
+   <profiles>
+      <profile>
+         <id>Copy-File</id>
+         <build>
+            <plugins>
+               <plugin>
+                  <groupId>io.fabric8</groupId>
+                  <artifactId>docker-maven-plugin</artifactId>
+                  <executions>
+                     <execution>
+                        <id>copy-from-container</id>
+                        <phase>generate-resources</phase>
+                        <goals>
+                           <goal>start</goal>
+                           <goal>copy</goal>
+                           <goal>stop</goal>
+                        </goals>
+                        <configuration>
+                           <images>
+                              <image>
+                                 <name>busybox:latest</name>
```

I believe we can omit default tag:

```suggestion
                                 <name>busybox</name>
```